e2e-tests/tests/gr-scaling/01-create-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 10
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      get_cr \
+        | yq eval '.spec.allowUnsafeConfigurations=true' - \
```

Comment:
Please specify `group replication` and all needed parameters for the test, because how the test is now created it is testing `async` because it is specified in functions to always use `async` by default regardless of what is specified in cr.yaml - https://github.com/percona/percona-server-mysql-operator/blob/main/e2e-tests/functions#L117
Although I think it would be best to remove that line in functions and every test should specify all the parameters needed for the test otherwise it is just making confusion.
Please do this change in this PR as well (so remove the line and add `async` or `group-replication` parameter to every test where needed).

Reply:
To expand the task a little bit please check the difference between `scaling` and `scaling-proxysql` tests for PXC. From what I see they are both testing `proxysql`, but my understanding is that `scaling` should be running with `haproxy`. (I might be missing something here, but let's check)

Reply:
@tplavcic as I remember in the scaling test we test only PXC scaling in scaling-proxysql we test proxysql only  :)

Reply:
@hors Ack, so we are probably missing haproxy scaling test and `scaling` and `scaling-proxysql` probably have a lot in common.